README.md

```diff
@@ -65,6 +84,21 @@ Replace `<something>` with the name of your S3 bucket, e.g. `wonderfulbali234`.
 }
 ```
 
+# CORS Policies
+
+A correctly configured bucket should have a [CORS policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html) attached to it that permits requests from the domain where your Wiki is hosted to access the thumbnails uploaded by this extension. If the domain where your Wiki is hosted is at `www.example.com`, attach the following policy:
```

Comment:
It shouldn't be worded like "all wikis need to set this".
CORS policy is not needed for 99% of wikis. Only for wikis that uses very specific JavaScript that downloads images like this.

Should name this section "Troubleshooting" (with subheader "my wiki usually shows thumbnails as popups, and now they don't work" or something), so that this is only recommended if the user actually has a problem.